unistd/dir.c

```diff
@@ -156,7 +156,7 @@ static int _resolve_abspath(char *path, char *result, int resolve_last_symlink,
 		}
 		else {
 			char *rem = slash;
-			while (*rem++ == '/')
+			while (*++rem == '/')
```

Comment:
after sleeping on it, maybe we should write it in "longer" form to be more readable:
```c
while (*rem == '/')
    rem++;
```

This seems less mind-bending for me :). But I'm leaving the decision up to You.

Reply:
I will go with your proposition as it is more clear and more DO-178 compliant. Thanks!